src/pykeen/nn/node_piece/representations.py

```diff
@@ -298,3 +315,43 @@ def _plain_forward(
             ),
             self.aggregation_index,
         )
+
+    def ratio_unique_hashes(self) -> RatioInfo:
+        """
+        Return the ratio of unique hashes in the total pool.
+
+        :return:
+            A ratio information tuple
+
+        .. todo:: @migalkin explain why you would want to calculate this
```

Comment:
@migalkin needs more high-level documentation about why someone would want to use this function

Reply:
done in https://github.com/pykeen/pykeen/pull/896/commits/2aed7771dc7e3d3376fb312bfaa3da4e80e8f3df